README.md

```diff
@@ -104,7 +104,10 @@ java -jar cas/build/libs/cas.war
 Run the CAS web application as an executable WAR via Spring Boot. This is most useful during development and testing.
 
 ```bash
-./gradlew[.bat] bootRun
```

Comment:
"./gradlew[.bat] bootRun" still appears, see the added part, I don't know why git specify it as deleted.